Modules/Tests/YosemiteTests/Model/Extensions/OrderStatsV4Interval+DateTests.swift

```diff
@@ -11,7 +11,9 @@ final class OrderStatsV4Interval_DateTests: XCTestCase {
                                             dateEnd: dateStringInSiteTimeZone,
                                             subtotals: mockIntervalSubtotals)
         // As long as the dates are parsed and formatted in the same time zone, they should be consistent.
-        let timeZone = TimeZone(secondsFromGMT: 29302)!
+        // Note that iOS 26 has changed behaviour here, and excess seconds in the timezone break the calculation
+        // This is 488 minutes, or 8 hours and 8 minutes
+        let timeZone = TimeZone(secondsFromGMT: 29280)!
```

Comment:
🤔 interesting

Reply:
It's weird. Do you think the logic is sound here? I wrapped myself up a bit yesterday trying to figure out which way round it was. It seems fine anyway...

Reply:
I'm merging it anyway, but we can come back to it depending what you think...